stdlib/public/Concurrency/Task.swift

```diff
@@ -15,22 +15,39 @@ import Swift
 
 // ==== Task -------------------------------------------------------------------
 
-/// An asynchronous task (just "Task" hereafter) is the analogue of a thread for
-/// asynchronous functions. All asynchronous functions run as part of some task.
+/// A unit of asynchronous work.
 ///
-/// A task can only be interacted with by code running "in" the task,
-/// by invoking the appropriate context sensitive static functions which operate
-/// on the "current" task. Because all such functions are `async` they can only
+/// All asynchronous functions run as part of some task.
+///
+/// Only code that's running as part of the task can interact with that task,
+/// by invoking the appropriate context-sensitive static functions which operate
+/// on the current task. Because all such functions are asynchronous, they can only
 /// be invoked as part of an existing task, and therefore are guaranteed to be
 /// effective.
```

Comment:
Sorry, this is outdated wording here; 

We have both static (non async) functions and instance functions now. The wording about async functions and properties is pretty ancient sorry.

I think we can remove the entire sentence from "Because all such functions are asynchronous ... " since those dont exist anymore and they're handled by the static and instance ones.